cEP-0016.md

```diff
@@ -0,0 +1,180 @@
+# Git Commit Content Inspection
+
+| Metadata |                                                   |
+| -------- | ------------------------------------------------- |
+| cEP      | 16                                                |
+| Version  | 1.0                                               |
+| Title    | Git commit content inspection                     |
+| Authors  | Kriti Rohillla <mailto:kritirohilla567@gmail.com> |
+| Status   | Implementation due                                |
+| Type     | Process                                           |
+
+## Abstract
+
+This document describes coala process rules which will be enforced by the
+GitCommitBear. Some new changes will be introduced in the implementation of
+GitCommitBear based on which the commits made to coala will be detected and
+verified.
+
+## Introduction
+
+The GitCommitBear at present performs a check on the content of regular commits
+made to coala. However, there are many special types of commit messages that
+should be used only in conjunction with patches containing a special type of
+content.
+
+For example, git revert commit which is used to revert any previous commit and
+git merge commit which is generated whenever someone performs a git rebase and
```

Comment:
a `git rebase` doesnt trigger a `git merge`.

You are probably talking about the GitHub "Rebase" button.  That is a valid example, as it is a major problem, but you need to properly understand it, and help the reader understand it.